api/src/certification/scoring/domain/models/CapacitySimulator.js

```diff
@@ -40,25 +44,21 @@ export class CapacitySimulator {
   }
 }
 
-export function findIntervalIndexFromScore({ score, scoringIntervalsLength, weights }) {
+// TODO move this to result
```

Comment:
Si on peut preciser le this (je comprends cette classe) ou deplacer ce todo pour pas qu'il soit seul :)